.github/workflows/main.yml

```diff
@@ -182,6 +182,11 @@ jobs:
     if: ${{ ! github.event.pull_request.draft || contains(github.event.pull_request.title, '[force-system-test]') }}
     runs-on: ubuntu-22.04
     timeout-minutes: 30
+    env:
+      EXPORT_AZ_CLIENT_ID: ${{ secrets.EXPORT_AZ_CLIENT_ID }}
+      EXPORT_AZ_CLIENT_PASSWORD: ${{ secrets.EXPORT_AZ_CLIENT_PASSWORD }}
+      EXPORT_AZ_TENANT_ID: ${{ secrets.EXPORT_AZ_TENANT_ID }}
+      EXPORT_AZ_KEY_VAULT_NAME: ${{ secrets.EXPORT_AZ_KEY_VAULT_NAME }}
```

Comment:
We should restrict the scope these are available I reckon. Does it work if we only set them for the steps where we run the tests?

Reply:
I don't see how that'd work if we need these env variables to be in several different docker containers

Reply:
Ah yes, good point